packages/xo-server-auth-oidc/index.js

```diff
@@ -118,16 +118,20 @@ class AuthOidc {
     }
 
     this.#unregisterPassportStrategy = xo.registerPassportStrategy(
-      new Strategy(conf, async (issuer, profile, done) => {
+      new Strategy(conf, async (issuer, profile, context, idToken, accessToken, refreshToken, params, done) => {
```

Comment:
`accessToken`, `refreshToken` and `params` don't apprear to be used. According to [passport-openidconnect](https://github.com/jaredhanson/passport-openidconnect/blob/master/lib/strategy.js#L232C67-L232C74), you can remove them and still get your `done` callback
```suggestion
      new Strategy(conf, async (issuer, profile, context, idToken, done) => {
```